app/src/views/OperationalLearning/Stats/OperationalLearningMap/index.tsx

```diff
@@ -0,0 +1,267 @@
+import {
+    useCallback,
+    useMemo,
+    useState,
+} from 'react';
+import {
+    Container,
+    NumberOutput,
+    TextOutput,
+} from '@ifrc-go/ui';
+import { useTranslation } from '@ifrc-go/ui/hooks';
+import { maxSafe } from '@ifrc-go/ui/utils';
+import {
+    _cs,
+    isDefined,
+    isNotDefined,
+    listToMap,
+} from '@togglecorp/fujs';
+import {
+    MapBounds,
+    MapLayer,
+    MapSource,
+} from '@togglecorp/re-map';
+import { type CirclePaint } from 'mapbox-gl';
+
+import GlobalMap from '#components/domain/GlobalMap';
+import Link from '#components/Link';
+import MapContainerWithDisclaimer from '#components/MapContainerWithDisclaimer';
+import MapPopup from '#components/MapPopup';
+import useCountry from '#hooks/domain/useCountry';
+import {
+    DEFAULT_MAP_PADDING,
+    DURATION_MAP_ZOOM,
+} from '#utils/constants';
+import { getCountryListBoundingBox } from '#utils/map';
+import { type GoApiResponse } from '#utils/restRequest';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+type OperationLearningStatsResponse = GoApiResponse<'/api/v2/ops-learning/stats/'>;
+const sourceOptions: mapboxgl.GeoJSONSourceRaw = {
+    type: 'geojson',
+};
+
+interface CountryProperties {
+    countryId: number;
+    name: string;
+    learningCount: number;
+}
+interface ClickedPoint {
+    feature: GeoJSON.Feature<GeoJSON.Point, CountryProperties>;
```

Comment:
If using GlobalMap, we should just save the property field and now the whole geojson feature.